core/sessions/user.go

```diff
@@ -10,17 +10,20 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 
 	"github.com/pkg/errors"
+	"gopkg.in/guregu/null.v4"
 )
 
 // User holds the credentials for API user.
 type User struct {
 	Email             string `gorm:"primary_key"`
 	HashedPassword    string
 	CreatedAt         time.Time `gorm:"index"`
-	TokenKey          string
-	TokenSalt         string
-	TokenHashedSecret string
+	TokenKey          null.String
+	TokenSalt         null.String
+	TokenHashedSecret null.String
 	UpdatedAt         time.Time
+
+	TokenSecret null.String // TODO: looks unused?
```

Comment:
Looks suspicious to me, I can't imagine why we'd ever want to persist this. If we drop the column and remove the field does anything break?

Reply:
Same here, it was a column in the db I had to add to the model to appease sqlx (it complains if you're scanning * to a model and are missing some fields by default unless you opt in to ignore them). So I don't think it was used anywhere